app/views/evaluation/import.html.haml

```diff
@@ -1,7 +1,13 @@
 %h1 Import evaluation data from PICA
+%body
+  - if flash[:error]
```

Comment:
This is mostly here because I had trouble figuring out how to properly use `flash[:errors].full_messages.each do |error|`.  That seems to be mainly for use with validators, which this error doesn't really fall under.  So as a workaround, I just saved the error as a string.  Any advice?